apps/investigation/kinds/investigation.cue

```diff
@@ -0,0 +1,25 @@
+package kinds
+
+// This is our Investigation definition, which contains metadata about the kind, and the kind's schema
+investigation: {
+	// Name is the human-readable name which is used for generated type names.
+	kind:  "Investigation"
+	scope: "Namespaced"
+	codegen: {
+		frontend: false
+		backend:  true
+	}
+	pluralName: "Investigations"
+	current: "v1-alpha"
+	versions: {
+		"v1-alpha": {
+			version: "v1-alpha"
```

Comment:
Versions should be alphanumeric. Also, you don't have to specify the version name, it's inferred from the key name:

```suggestion
	current: "v1alpha1"
	versions: {
		"v1alpha1": {
```